charmcraft/utils/project.py

```diff
@@ -24,7 +24,6 @@
 from craft_cli import emit
 from jinja2 import Environment, FileSystemLoader, PackageLoader, StrictUndefined
 
-import charmcraft
```

Comment:
I almost left a comment from an [earlier commit](https://github.com/canonical/charmcraft/pull/1340/commits/ddd361738b46a28329361a478a496ac262365c0e#diff-dd1802bfa45942a4028f22581575ecce25ece667160712c4ce35bc723a00688bR27) but decided to wait to see if a subsequent commit would use this import